software/SolarAISVN/SolarAISVN.ino

```diff
@@ -25,14 +25,21 @@ uint64_t uS_TO_S_FACTOR = 1000000;  // Conversion factor for micro seconds to se
 uint64_t TIME_TO_SLEEP = 120;
 
 int adcValue = 0;
+int batValue = 0;
 
 void setup() {
+  pinMode(LED_BUILTIN, OUTPUT);
+  digitalWrite(LED_BUILTIN, LOW);  
   Serial.begin(115200); 
-  delay(2000);
+  delay(50);
+  digitalWrite(LED_BUILTIN, HIGH);   
+  delay(1000);
```

Comment:
Just a short blink added, not 2 seconds of light

Reply:
Good optical feedback.